src/main/java/com/hazelcast/kubernetes/KubernetesApiEndpointResolver.java

```diff
@@ -56,6 +61,9 @@
             return getSimpleDiscoveryNodes(client.endpointsByName(serviceName));
         } else if (serviceLabel != null && !serviceLabel.isEmpty()) {
             return getSimpleDiscoveryNodes(client.endpointsByLabel(serviceLabel, serviceLabelValue));
+        } else if (podLabel != null && !podLabel.isEmpty()) {
+            logger.warning("Using Pod label to discover nodes...");
```

Comment:
1. Why is it at the "warning" logging level? I think it should be "debug" (maybe "info", but then if you try it, it logs very frequently, so better "debug").
2. Also, I like the logging, could you add it for other cases (2 lines above, 4 lines above)?

Reply:
1. Actually, I added this log to check the `else if` block actually runs from pods' logs then I forgot to delete 🙂Unfortunately, our [ILogger](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/logging/ILogger.java) class does not have `debug` level so I will use `fine`. Is it ok for you?
2. Added 👍